### PR TITLE
fix: allow to use custom PKGS_PREFIX

### PR DESCRIPTION
**Pull request by aarnaud**
Allow to use custom PKGS_PREFIX when building a custom kernel or kernel module

### CODE REVIEW

**Comment by frezbo** [comment]
The Makefile here is autogenerated, you'd need to update kres.yaml with this and run `make rekres`

```diff
diff --git a/.kres.yaml b/.kres.yaml
index 8744f0e..021ea42 100644
--- a/.kres.yaml
+++ b/.kres.yaml
@@ -41,6 +41,7 @@ spec:
   extraBuildArgs:
     - TAG
     - PKGS
+    - PKGS_PREFIX
   useBldrPkgTagResolver: true
 ---
 kind: common.Build
@@ -78,6 +79,8 @@ spec:
         defaultValue: $(REGISTRY_AND_USERNAME)/extensions:$(TAG)
       - name: PKGS
         defaultValue: v1.7.0-alpha.0-12-g583e519
+      - name: PKGS_PREFIX
+        defaultValue: ghcr.io/siderolabs
     depends:
       - internal/extensions/descriptions.yaml
     script:

```

**Comment by aarnaud** [comment]
Hi @frezbo, thanks for your feedback,  I did the changes.

File: .kres.yaml [hunk 1]
```diff
@@ -78,6 +79,8 @@ spec:
         defaultValue: $(REGISTRY_AND_USERNAME)/extensions:$(TAG)
       - name: PKGS
         defaultValue: v1.7.0-alpha.0-13-gf376a53
+      - name: PKGS_PREFIX
+        defaultValue: $(REGISTRY_AND_USERNAME)
```
Comment (smira) [inline .kres.yaml:83]: `ghcr.io/siderolabs`

Comment (aarnaud) [reply .kres.yaml]: done,

**Review by frezbo — APPROVED** [review]

**Pull request merged**